Framework/Algorithms/inc/MantidAlgorithms/PolarizationCorrections/SpinStateValidator.h

```diff
@@ -34,6 +35,13 @@ class MANTID_ALGORITHMS_DLL SpinStateValidator : public Kernel::TypedValidator<s
   static const std::string ZERO;
   static const std::string ONE;
 
+  static const std::string MINUS_PLUS;
```

Comment:
It's great that we've re-defined these constants in the `PolarizationCorrectionsHelper` so that this validator is more generic. Since that change, I can't see anywhere that these spin state constants and the flipper config constants above are used now - is it possible to remove them?